fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PruneProjectRule.java

```diff
@@ -20,7 +19,6 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 public class PruneProjectRule extends TransformationRule {
     public PruneProjectRule() {
```

Comment:
The rule seems used to fill a col to avoid empty projection. But why should we need this? If other prune rules casusd the empty projection, I think we should repair other rules.  Am I right? @kangkaisen @Seaven